website/common/script/ops/buy/buyMount.js

```diff
@@ -41,6 +42,7 @@ export class BuyHourglassMountOperation extends AbstractHourglassItemOperation {
     if (user.markModified) user.markModified('items.mounts');
 
     this.subtractCurrency(user);
+    removeItemByPath(user, `timeTravelStable.mounts.${this.key}`);
```

Comment:
at first I thought we'd need to use `getItemInfo` which would have the path already, but on the other hand we'd create this object just to get the path out of it

@paglias what do you think ?

Reply:
I think that using `getItemInfo` would be more future proof in case the path changes